### PR TITLE
Ltfas bebeco

### PR DESCRIPTION
**Pull request by faselm**
Commits until 7e3d07c were already on master when I checked out the branch

### CODE REVIEW

**Comment by ltclm** [comment]
@faselm  i have removed the first commits of this pr.
* can you please prepare a testlink
* please squash your commits and add them with the correct username

**Comment by faselm** [comment]
@ltclm Here is the test link: https://mf-chsdi3.dev.bgdi.ch/ltfas_bebeco/shorten/8c0f7ac0e0

I will squash all this!

File: chsdi/models/vector/vbs.py [hunk 1]
```diff
@@ -160,14 +160,24 @@ class BundestankstellenBebeco:
     __table_args__ = ({'schema': 'militaer', 'autoload': False, 'extend_existing': True})
     __template__ = 'templates/htmlpopup/bundestankstellen.mako'
     __bodId__ = 'ch.vbs.bundestankstellen-bebeco'
-    __queryable_attributes__ = ['ort', 'plz', 'strasse']
+    __queryable_attributes__ = ['ort', 'standort', 'adresse', 'plz', 'produkt_de', 'produkt_fr', 'produkt_it']
```
Comment (ltclm) [inline chsdi/models/vector/vbs.py:163]: the list of queryable_attributes is linked to the sphinxsearch configuration [1]
please use this new list of attributes to the sphinxsearch configuration:
https://github.com/geoadmin/service-sphinxsearch/blob/ac91448e203f4f5b916c5d4abc9263f088b109f1/conf/vbs.conf.part#L82

[1] https://ltwiki.adr.admin.ch:8443/display/BGDI/03_VIEWER

**Review by ltclm — APPROVED** [review]
pr in service-sphinxsearch is needed.
the rest looks good to me, thank you!

File: chsdi/models/vector/vbs.py (hunk 1 above)
Comment (faselm) [reply chsdi/models/vector/vbs.py:163]: But produkt_de, produkt_fr, produkt_it are searchable. There are also in the pr in the sphinxsearch configuration... https://github.com/geoadmin/service-sphinxsearch/pull/433

Comment (ltclm) [reply chsdi/models/vector/vbs.py:163]: oh, i forgot that there's already a sphinxsearch pr. its all good then.

**Review by b4l — APPROVED** [review]
Looks good!

**Review by rebert — APPROVED** [review]

**Pull request merged**